config/credentials.yml.enc

```diff
@@ -1 +1 @@
-ElAZomVweQaVbuQyaANRHJ3EYmyt6GTkqEJgabzwRuhTskIgFBk8Vf2oT2L1mta34RSlWUh1vKg/U83Ht21QK4b6KbrpWwMGxTvjk3tdyN4OIJntMnWYDZaGlyMbhtdTTc0XsdglMXwu+cjBEXfaGBqUDEcm2F4B/V4fCLgUztCFbK9TJiKroHs/jRwYT3pLk+BPlxz3xZat6NuBCiXDyIPQxMqfQ7axEhcjCJ9C3XsU2Stjht5wf2OE0/BRYnFb4J4GEWl8NM9mKM6KUYBaEudclD/frrAhTTu6MU4RKm6zyOYNE7tBknOd0shj9MILFVu/swBXHZSqU/GUlfpnJJVvMVV8ccBwGbK/b54ge9mJzBcp76jSFdaR+fm7DBEMCttZvLv+tjcpTErUn/V2E4556LmatsYqL816sbCH4GdyM5wSCHc1Cr2U/Oib4DAYKix+evk/RX92uvSTf2hFmjULnGyX2iA8wwPC2YLWiHHKzRn6A088QprGCv3Y0eWbaTAT6+hcLjGLD9rPQODVQQpugp4uG4oVKdU3TqVmWY3kfIv6VvxdgSmh8Z6tNiWtWBqtDxTZgE77Y7qp2SKxEPdGV8wVmuCFcyir2dvmKvbTrocciw==--QFSK5TTtkVVGjPIq--XdqLsVo5zugUppyGQuS2UA==
\ No newline at end of file
+uC9/RIpwqP1Z2eK5hPSbrfntxZCUUAeMwESr3II41CgC1a1mqgKvAZR7qf/54+G19bUo4BEXMUOsTLRuPSIQzdb/epbB4X5JSjC+hSzZjpDvCBfZCFRmJBKo2+eyzGSYh+ltKlTTi4rW9pSD6DYn25Kz7fYPoTcShIjdnyErg79IaOBQ+bH5pNRcKCiIRtzcd5DnJYf1L7fsAb9419XxJRtrMmAAEYA2+xUpoHq9ETrv6OjP8TyK1GeSTQV83k/MYD/WFyQ+lckIvc2qWiN7PDe8yCO1xyOei1OMZ8cmh39lk/OZX+Fbu82H25CB5JoM4jSoTzrP8yiyimppopVoNx/ykjztB3eX0tZlh7HvbxEk3gV+qPLFyjenObSK44V9akeDxLij0L8DeooZSP0TCqP7nLUgfxnnI+c+Mh1Y6jrVgIMQ8fFb8L1thokfIMfsTONCoxWbAbzDejzFPYrk5RTAfpXO4iLOfpvfl+BmxcjIcvKP8crM79LF7KQabjyDYw50IFg4bhiAuLASkRO4Vp43GEg4JmDSX9H5VHklN9UEseVBlm6n0w4CCM+fwN8L0CAmoCTkBqh2yiui2PXuY2mx8uzOUjjgqYXE+F8me3KB3B996tMWh8BHbiylG6UdEPG77DTxuJpAbl4sAzexst6h0O++hbuG3ml25YwIQ313Ll6jMheBMLPVF/oIWsIUh2TPYlYOf75zBbcS6KS7GDOFjzs9Cg4sINjYS/D7Pe5ZHswr8uNduXwAOBW5doTxpzxydlqpxXeUehuHTQVW0eH9y8VcLkX6qYlHKcX4--4jFefjNJJtJwipI6--2JEL86t169fr3OQFES3x0Q==
```

Comment:
🧐 Interesting, this file does not usually change, I am not saying that is wrong, I am just curious.